recipes/libigl/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class LibiglConan(ConanFile):
+    name = "libigl"
+    description = ("A Massively Spiffy Yet Delicately Unobtrusive Compression Library "
+                  "(Also Free, Not to Mention Unencumbered by Patents)")
+    topics = ("conan", "libigl", "geometry", "matrices", "algorithms")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libigl.github.io/"
+    license = "MPL-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    requires = ("eigen/3.3.9")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+ 
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["LIBIGL_USE_STATIC_LIBRARY"] = not self.options.shared
```

Comment:
I think setting this https://github.com/libigl/libigl/blob/7342dbe1f8bcc22cd09bbad249713e19aad28159/CMakeLists.txt#L25
to `"ON"` will give use the Conan install
https://github.com/libigl/libigl/blob/36e8435a2f724e83e14f79d128102d06b514a4f4/cmake/libigl.cmake#L487

```suggestion
        cmake.definitions["LIBIGL_USE_STATIC_LIBRARY"] = not self.options.shared
        cmake.definitions["LIBIGL_EXPORT_TARGETS"] = True
```